R/colour-text.r

```diff
@@ -1,8 +1,9 @@
-
+# fix the check NOTE:Namespace in Imports field not imported from: ‘crayon’
```

Comment:
I just checked: the NOTE is absolutely not about the code. It is due to the fact that crayon is listed in the Imports, but nothing was imported. To fix the note, I located in the code where crayon was used, imported the appropriate symbols, and used them directly just to make the code look nicer.